build-pipelines-PR.yml

```diff
@@ -51,7 +40,7 @@ resources:
     ref: refs/heads/main
 jobs:
 - job: Phase_1
-  displayName: Run self-contained build (binaries V1, where the binaries are all bundled together)
+  displayName: Run PR build for self contained
```

Comment:
nit and maybe just my personal reference but i like having the link between self contained and binaries V1. If possible I liked previous display name better. okay to update display name if we add a comment as well.

Reply:
I changed it back, since it was long, and it was cutting off in the ADO UI that's why made it short.